src/main/java/org/example/utility/UtilityMethods.java

```diff
@@ -24,4 +26,14 @@ public static boolean isEven(int a) {
     public static boolean isOdd(int a) {
         return !isEven(a);
     }
+
+    public static int countOdd(int[] array) {
+        checkNotNull(array);
```

Comment:
checkNotNull also accepts the second parameter - method. Please, add some informative message